components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -377,4 +377,14 @@ - (void)bottomDrawerContainerViewControllerDidChangeYOffset:
   }
 }
 
+- (void)
+    bottomDrawerContainerViewControllerDidUpdateValueForScrollViewIsScrolledToEndOfContent:
+        (MDCBottomDrawerContainerViewController *)containerViewController
+                                                        scrollViewIsScrolledToEndOfContent:
+                                                            (BOOL)
+                                                                scrollViewIsScrolledToEndOfContent {
+  UIColor *normalScrimColor = self.scrimColor ?: [UIColor colorWithWhite:0 alpha:(CGFloat)0.32];
```

Comment:
may be worth commenting here on why we added this behavior and what its trying to solve. so future contributors will better understand.